winch/filetests/filetests/x64/unreachable/with_spilled_local.wat

```diff
@@ -19,7 +19,6 @@
 ;;   1a:	 4883ec04             	sub	rsp, 4
 ;;   1e:	 44891c24             	mov	dword ptr [rsp], r11d
 ;;   22:	 0f0b                 	ud2	
-;;   24:	 4883c404             	add	rsp, 4
-;;   28:	 4883c410             	add	rsp, 0x10
-;;   2c:	 5d                   	pop	rbp
-;;   2d:	 c3                   	ret	
+;;   24:	 4883c410             	add	rsp, 0x10
+;;   28:	 5d                   	pop	rbp
+;;   29:	 c3                   	ret	
```

Comment:
I wonder if it would be worth omitting the offsets (except for jump targets) so that the diff would have shown only the `add rsp, 4` instruction going away?

Reply:
Can you clarify a bit what do you mean with this? More concretely, are you referring  to how the disassembly is formatted?

Reply:
The instructions in the original program from  offset 28 on are the same as those in the new program from 24 on, and the only thing that changed was the offsets. If we didn't emit the offsets, or selectively emitted them only for jump targets, the diff would show that the only thing missing was `add rsp, 4`.

Reply:
Oh right -- sorry for the confusion, I see it now. I think that's a good idea, it it has the potential to reduce verbosity. Are you good if I create a follow up for this? Because If I'm not wrong, this will change a good chunk of the filetests.

Reply:
Definitely not necessary to fix that in this PR, just something I was thinking would help future reviews :)